pkg/plugins/trivy/plugin.go

```diff
@@ -974,8 +976,6 @@ func (p *plugin) getPodSpecForClientServerMode(ctx trivyoperator.PluginContext,
 				Value: "true",
 			})
 		}
-		volumeMounts = append(volumeMounts, getScanResultVolumeMount())
-		volumes = append(volumes, getScanResultVolume())
 
 		if config.IgnoreFileExists() {
 			volumes = append(volumes, corev1.Volume{
```

Comment:
you probably want to move this out of the loop as well.

Reply:
you mean  the `IgnoreFileExists()` part ? it was there before, you think we should move it out ?